setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore = E501
```

Comment:
Please do this in the .flake8 file instead.

Reply:
Ah sorry, `ls` doesn't show `.*` so I missed it :smile:.